### DISCUSSION TITLE
Abogabot

### DISCUSSION THREAD

**Discussion by kerizrh** [Launch-X-Latam/MisionFrontEnd#223] [Show and tell]
Hola, a todos, espero que se encuentren con bien.
Veo que mucho han subido sus avances de la actividad de abogabot  y acá está el mío, me gustaría que me dieran sus críticas constructivas. 
Hice una versión para escritorio y otra móvil, trabajé mayormente con Figma.
También estuve haciendo un poco de prueba y error con HTML y CSS para intentar recrearla en la web.
 
👇
[Abogabot by Keriz](https://github.com/Kerizr/Ejercicio-de-UI-UX-Abogabot-)

@romarpla

**Comment by doguedogue** [comment]
Se ve muy bien!

**Reply by kerizrh** [reply]
¡Gracias!
